library/modules/bigip_software_install.py

```diff
@@ -40,10 +45,21 @@
       - activated
       - installed
     default: activated
+  type:
+    description:
+      - The type of the BIG-IP.
+      - Defaults to C(standard), the other choice is C(vcmp).
+    type: str
+    default: standard
+    choices:
+      - standard
+      - vcmp
+  version_added: "f5_modules 1.2"
```

Comment:
this is in the wrong spot make this in line with 'description' this is probably causing the sanity tests to collapse

Reply:
Ah! I don't know why I didn't even think this was the issue! Thanks a lot @wojtek0806 😄